### PR TITLE
[doc] Proofread debugging.md

### PR DESCRIPTION
**Pull request by k-ye**
### Purpose

- No issue is linked to this PR.

### Changes

- No changes.

### Review Instructions

- No instructions.

### CODE REVIEW

File: website/docs/lang/articles/misc/debugging.md [hunk 1]
```diff
@@ -201,10 +204,8 @@ def copy(dst: ti.template(), src: ti.template()):
 
 ## Pretty Taichi-scope traceback
```
Comment (k-ye) [inline website/docs/lang/articles/misc/debugging.md:205]: I can't say intentionally hiding stack frames is a nice thing to have. This has also resulted in `_taichi_skip_traceback = x` being scattered all over Taichi's codebase. In the future we should consider simplifying Taichi's Python code base, and removing this exception hook hack.

Comment (rexwangcc) [reply website/docs/lang/articles/misc/debugging.md:207]: Strongly agree, would you mind creating an issue somewhere to track this so we don't forget about it in the future?

File: website/docs/lang/articles/misc/debugging.md [hunk 2]
```diff
@@ -270,17 +271,14 @@ Traceback (most recent call last):
...
-`__call__`'s. These are the Taichi internal stack frames. They have
-almost no benefit for end-users but make the traceback hard to read.
-
-For this purpose, we may want to use `ti.init(excepthook=True)`, which
-_hooks_ on the exception handler, and make the stack traceback from
-Taichi-scope easier to read and intuitive. e.g.:
+Many of the stack frames are the Taichi compiler implementation details, which
+could be noisy to read. You could choose to elide them by enabling
+`ti.init(excepthook=True)`, which _hooks_ on the exception handler, and makes
+the stack traceback from Taichi-scope more intuitive.
```
Comment (rexwangcc) [inline website/docs/lang/articles/misc/debugging.md]: +100

**Review by rexwangcc — APPROVED** [review]
LGTM, thanks! (I committed a few very minor updates.)

File: website/docs/lang/articles/misc/debugging.md (hunk 1 above)
Comment (k-ye) [reply website/docs/lang/articles/misc/debugging.md:207]: Filed https://github.com/taichi-dev/taichi/issues/2489 to track this issue.

**Pull request merged**